### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by james-munson**
Removes max draw distance of ground markers
Removes pkvison plugin (replaced by updates to player indicator)
Adds Inventory Viewer config option to hide viewer when player inventory is open
Adds grouping option to Inventory Viewer
Adds whether to show a label with the free slots in the inventory
Adds vanguard hp tracker
Adds ammo plugin
Adds BA team health bars
Adds world hop history
Adds show what you hit next to the XP drop above enemies head
Adds a toggle to only show player indicators in the wilderness. 
Adds player indicator for attackable players
Adds attacker color indicator
Adds player indicator config to show offline friends
Adds a highlight player option to the player indicators which will show highlighted player name as well as show highlighted players target name to player indicator
Adds combat levels next to player names to player indicator
Adds a filter for player indicator based on your level. feature-request

### CODE REVIEW

**Comment by gazivodag** [comment]
Will follow up with you to review these changes before approving

File: runelite-client/src/main/java/net/runelite/client/plugins/ammo/AmmoPlugin.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import net.runelite.client.eventbus.Subscribe;
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.ui.overlay.infobox.InfoBoxManager;
+
+@PluginDescriptor(
+	name = "Ammo",
+	description = "Shows the current ammo the player has equipped",
+	tags = {"bolts", "darts", "chinchompa"}
```
Comment (zeruth) [inline runelite-client/src/main/java/net/runelite/client/plugins/ammo/AmmoPlugin.java]:
```suggestion
tags = {"bolts", "darts", "chinchompa"},
type = utility
```

File: runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java [hunk 2]
```diff
@@ -27,20 +27,31 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import net.runelite.api.events.ConfigChanged;
+import net.runelite.api.events.InteractChanged;
+import net.runelite.client.util.WildcardMatcher;
 
 @PluginDescriptor(
 	name = "Player Indicators",
```
Comment (zeruth) [inline runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java]:
```suggestion
	name = "Player Indicators",
	description = "Highlight players on-screen and/or on the minimap",
tags = {"highlight", "minimap", "overlay", "players"},
type = "utility"
```

File: runelite-client/src/main/java/net/runelite/client/plugins/vanguards/VanguardPlugin.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import javax.inject.Inject;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.ui.overlay.OverlayManager;
+
+@PluginDescriptor(
+        name= "Vanguard HP Overlay",
+        description= "tracks HP of all three vanguards",
+        tags= {"overlay", "vangs", "cox"},
+        enabledByDefault = false
```
Comment (zeruth) [inline runelite-client/src/main/java/net/runelite/client/plugins/vanguards/VanguardPlugin.java]:
```suggestion
     enabledByDefault = false,
type = "PVM"
```

**Comment by zeruth** [comment]
After those changes are made, this looks good to me :)

**Pull request merged**